setup.py

```diff
@@ -13,9 +21,9 @@ def readme():
       long_description=readme(),
       long_description_content_type="text/markdown",
       keywords='Python PDFium',
-      url='https://github.com/YinlinHu/pypdfium',
-      author='Yinlin Hu',
-      author_email='huyinlin@gmail.com',
+      url='https://github.com/mara004/pypdfium',
+      maintainer='Manuel Geißer',
+      maintainer_email='geisserml@gmail.com',
```

Comment:
Are you sure you want to include url and maintainer changes in the pull request?

Reply:
I did this in my repository, not thinking of the pull request